server/src/routes/api/v1/userActivityRouter.js

```diff
@@ -0,0 +1,28 @@
+import express from "express"
+
+import { User } from "../../../models/index.js"
+
+const userActivityRouter = new express.Router({ mergeParams: true })
+
+userActivityRouter.get("/", async (req, res) => {
+    const userData = req.user
+
+    try {
+        const user = await User.query().findById(userData.id)
+        user.salads = await user.$relatedQuery("salads")
+        user.reviews = await user.$relatedQuery("reviews")
+        const userSaladsSorted = user.salads.sort((b,a) => {
+            return Date.parse(a.createdAt) - Date.parse(b.createdAt)
+        })
```

Comment:
use `orderBy` chainable method to handle this through the database query
https://vincit.github.io/objection.js/api/query-builder/find-methods.html#orderby